command/backup/backup.go

```diff
@@ -0,0 +1,155 @@
+package backup
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-sdk/archive"
+	"github.com/0xPolygon/polygon-sdk/command/helper"
+	"github.com/0xPolygon/polygon-sdk/types"
+	"github.com/hashicorp/go-hclog"
+)
+
+type BackupCommand struct {
+	helper.Base
+	Formatter *helper.FormatterFlag
+	GRPC      *helper.GRPCFlag
+}
+
+// DefineFlags defines the command flags
+func (c *BackupCommand) DefineFlags() {
+	c.Base.DefineFlags(c.Formatter, c.GRPC)
+
+	c.FlagMap["data"] = helper.FlagDescriptor{
+		Description: "The path of backup data to save",
+		Arguments: []string{
+			"BACKUP_FILE",
+		},
+		ArgumentsOptional: false,
+	}
+
+	c.FlagMap["from"] = helper.FlagDescriptor{
+		Description: "Begining height of chain in backup",
+		Arguments: []string{
+			"FROM",
+		},
+		ArgumentsOptional: true,
+	}
+
+	c.FlagMap["to"] = helper.FlagDescriptor{
+		Description: "End height of the chain in backup",
+		Arguments: []string{
+			"TO",
+		},
+		ArgumentsOptional: true,
+	}
+}
+
+// GetHelperText returns a simple description of the command
+func (c *BackupCommand) GetHelperText() string {
+	return "Create blockchain backup data by fetching from running node"
+}
+
+func (c *BackupCommand) GetBaseCommand() string {
+	return "backup"
+}
+
+// Help implements the cli.Command interface
+func (c *BackupCommand) Help() string {
+	c.DefineFlags()
+
+	return helper.GenerateHelp(c.Synopsis(), helper.GenerateUsage(c.GetBaseCommand(), c.FlagMap), c.FlagMap)
+}
+
+// Synopsis implements the cli.Command interface
+func (c *BackupCommand) Synopsis() string {
+	return c.GetHelperText()
+}
+
+// Run implements the cli.Command interface
+func (c *BackupCommand) Run(args []string) int {
+	flags := c.Base.NewFlagSet(c.GetBaseCommand(), c.Formatter, c.GRPC)
+
+	var out, rawFrom, rawTo string
+	flags.StringVar(&out, "out", "", "")
+	flags.StringVar(&rawFrom, "from", "0", "")
+	flags.StringVar(&rawTo, "to", "", "")
```

Comment:
You use `out` here, but in the flags you define `data`

Reply:
Thank you! I fixed at 1f700bf72068dda1e41c5f022f0c54b95d9def40